developer-docs-site/docs/tutorials/full-node/network-identity-fullnode.md

```diff
@@ -23,7 +33,7 @@ Docker container is currently supported only on Linux x86-64 platform. If you ar
 
 ## Creating a static identity for a FullNode
 
-FullNodes will automatically start up with a randomly generated network identity (a `PeerId` and a public key pair). This works well for regular FullNodes, but you may wish to be added to another node's allowlist, provide specific permissions or run your FullNode with the same identity. In this case, creating a static network identity can help.
+To create a static identity for your FullNode, you should first create a private key public key pair for your FullNode. You then use the public key in the `peer_id` field in your `public_full_node.yaml` to create a static network identity for your FullNode.
```

Comment:
```suggestion
To create a static identity for your FullNode, you should first create a private key, public key pair for your FullNode. You then use the public key in the `peer_id` field in your `public_full_node.yaml` to create a static network identity for your FullNode.
```

> You then use the public key in the `peer_id` field in your `public_full_node.yaml` to create a static network identity for your FullNode.

This isn't correct, the peer_id is derived from the public key.

Reply:
Thanks for catching this. I also added an additional command using Aptos CLI showing how to extract peer. Sorry about this churn.